### PR TITLE
Remove deletion suggestion from invalid_iaas_credentials.json

### PR DESCRIPTION
**Pull request by tnierman**
Rewords invalid_iaas_credentials message

### CODE REVIEW

**Comment by clcollins** [comment]
LGTM, but need mgmt approval too.

File: osd/invalid_iaas_credentials.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
     "service_name": "SREManualAction",
     "cluster_uuid": "${CLUSTER_UUID}",
     "summary": "Action required: delete inaccessible cluster",
```
Comment (c-e-brumm) [inline osd/invalid_iaas_credentials.json:5]: Should update this field also to remove the delete reference.

"Action required: Restore missing cloud credentials"

File: osd/invalid_iaas_credentials.json [hunk 2]
```diff
@@ -3,6 +3,6 @@
...
-    "description": "Your cluster requires you to take action because Red Hat OpenShift Cluster Manager is not able to access the infrastructure with the provided credentials. If you intended to terminate this cluster then please delete the cluster, otherwise file a support request.",
+    "description": "Your cluster requires you to take action because Red Hat SRE is not able to access the infrastructure with the provided credentials. Please restore the credentials and permissions provided during install, or file a support request.",
```
Comment (c-e-brumm) [inline osd/invalid_iaas_credentials.json:6]: Should remove "SRE" if we're changing from OpenShift Cluster Manager. It doesn't matter what can't use the infrastructure and avoid references to humans unless it's relevant.

**Pull request merged**